packages/shared/lib/core/wallet/utils/outputs/outputIdFromTransactionData.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Returns the outputId from transation id and output index.
+ * @param transactionId The id of the transaction.
+ * @param outputIndex The index of the output.
+ * @returns The output id.
+ */
+export function outputIdFromTransactionData(transactionId: string, outputIndex: number): string {
+    return transactionId + uInt16NumberToLittleEndianHex(outputIndex)
+}
+
+function uInt16NumberToLittleEndianHex(num: number) {
```

Comment:
This could be a more generally accessible utils function in the `utils.ts` file (we still have yet to create a `utils` core module), it might be used somewhere else in the future?